docs/projects/hello-world/_posts/2019-09-04-erlang.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Hello world in Erlang
+layout: default
+last-modified: 2019-09-04
+featured-image: hello-world-in-erlang-featured-image.JPEG
+tags: [erlang, hello-world]
+authors:
+  - the_renegade_coder
```

Comment:
Since you're the primary author, you can pull me off this (or at least swap the order). 😃